generate-profile.lic

```diff
@@ -0,0 +1,120 @@
+=begin
+This script takes partial YAML from profile-gen/ and resolves all %INCLUDE
```

Comment:
Do you plan to document this on elanthipedia?

Reply:
I wasn't particularly planning on it, but I can if we start getting questions.  Right now I'm considering this an "advanced" option and I'm not sure I want to point people towards it.